packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -51,4 +51,31 @@ See [our README](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/t
 | `<dds-legal-nav>`        | `data-autoid="dds--legal-nav"`         | Component   |
 | `<dds-legal-nav-cookie-preferences-placeholder>` | `data-autoid="dds--privacy-cp"`        | Component   |
 
+### Language selector
+
+The option to use a language selector is available in lieu of the locale
+button/selector.
+
+The following attributes are needed:
+
+| Attribute                | Description                            |
+| ------------------------ | -------------------------------------- |
+| `language-selector-label`| `The input box placeholder`            |
+| `selected-language`      | `The initial selected language`        |
```

Comment:
Just checking, isn't the localeList also necessary? Might help to also show what the expected data structure should be.

Reply:
Hm, yeah I was under the impression `localeList` would always be passed down regardless of `languageSelector` being on or not for the `locale-button`, but that component is also optional in the first place. Let me update